### PR TITLE
ability to animate text

### LINKED ISSUE #161: Draw a text
**Is your feature request related to a problem? Please explain.**
It's not really a problem but it would be nice to have something more visual appealing.

**Describe the solution you'd like**
I would like to have some kind of interesting appearance of text from left to right. Based on #43 it is currently possible to let text of everything else for that matter fade in. Would be awesome to have `appear(:draw)` or something which only works for `text` at first and might make sense for everything else later.

**Describe alternatives you've considered**
Currently there is only the way to `fade` or `scale` but it doesn't show that text is from left to right (at least in English :smile: )

**Additional context**
Can be seen in manim here:
https://youtu.be/fNk_zzaMoSs?t=6 
Easier to see with 0.25x speed. It doesn't have to be this animation. A simple one like appearing from left to right using clipping might be a good start.

### PR DESCRIPTION
**Pull request by Wikunia**
## PR Checklist

If you are contributing to `Javis.jl`, please make sure you are able to check off each item on this list:

- [x] Did I update `CHANGELOG.md` with whatever changes/features I added with this PR?
- [x] Did I update the `TODO.md` (if applicable)?
- [x] Did I make sure to only change the part of the file where I introduced a new change/feature?
- [ ] Did I cover all corner cases to be close to 100% test coverage (if applicable)?
- [x] Did I properly add Javis dependencies to the `Project.toml` + set an upper bound of the dependency (if applicable)?
- [x] Did I properly add test dependencies to the `test` directory (if applicable)?
- [ ] Did I check relevant tutorials that may be affected by changes in this PR?
  - Might be possible to integrate this into the Jarvis tutorial
  - or into the PeriodicTable one @TheCedarPrince Like showing the element in big above the circle?
- [x] Did I clearly articulate why this PR was made the way it was and how it was made?

**Link to relevant issue(s)**
Closes #161 

**How did you address these issues with this PR? What methods did you use?**
Text can now be animated using the `appear` function and the symbol `:draw_text`. 
Only needed to override the `text` function and added one `appear` and one `disappear` function.
```julia
using Javis, Animations

function ground(args...)
    background("black")
    sethue("white")
    fontsize(20)
end

video = Video(400, 300)

javis(
    video,
    [
        BackgroundAction(1:100, ground),
        BackgroundAction(1:100, (args...) -> fontsize(30)),
        Action(
            1:100,
            (args...) -> text("Hello Stream!", -50, 50; halign = :center);
            subactions = [
                SubAction(1:15, sineio(), appear(:draw_text)),
                SubAction(76:100, sineio(), disappear(:draw_text)),
            ],
        ),
    ],
    tempdirectory = "current/images",
)
```

text

### CODE REVIEW

**Comment by Wikunia** [comment]
This also need some docstrings!

**Comment by Wikunia** [comment]
This fails because text is displayed differently on different operating systems. We might wanna write some kind of unit test instead and check that it just doesn't throw an error.

File: src/luxor_overrides.jl [hunk 1]
```diff
@@ -148,3 +148,89 @@ function scaleto(x, y)
...
 end
+
+function animate_text(
+    str,
+    pos::Point,
+    valign::Symbol,
+    halign::Symbol,
+    angle::Float64,
+    t::Float64,
+)
```
Comment (TheCedarPrince) [inline src/luxor_overrides.jl:159]: Missing a docstring here -- I see a lot of the code came directly from Luxor. It seems like there is enough of a change to merit a docstring for the user.

**Review by TheCedarPrince — APPROVED** [review]
Passed all tests on my end. I think you are good to go here after fixing up a missing docsstring

File: src/luxor_overrides.jl (hunk 1 above)
Comment (Wikunia) [reply src/luxor_overrides.jl:159]: This is an internal function though. I'll add a short developer docstring.

**Review by TheCedarPrince — APPROVED** [review]
Thanks for fixing up the docstring some more. I think it looks much better and ready for merging. Great work @Wikunia!

**Pull request merged**